ansible/roles/debops.postldap/defaults/main.yml

```diff
@@ -0,0 +1,627 @@
+                                                ansible_local.ldap.uri|d())
+                                            else [""]) | first }}'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_server_port [[[
+#
+# The TCP port which should be used for connections to the LDAP server.
+postldap__ldap_server_port: '{{ ansible_local.ldap.port
+                                            if (ansible_local|d() and ansible_local.ldap|d() and
+                                                ansible_local.ldap.port|d())
+                                            else ("389" if postldap__ldap_start_tls|bool else "636") }}'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_start_tls [[[
+#
+# If ``True``, Postfix will use STARTTLS extension to make encrypted
+# connections to the LDAP server.
+postldap__ldap_start_tls: '{{ ansible_local.ldap.start_tls
+                                          if (ansible_local|d() and ansible_local.ldap|d() and
+                                              (ansible_local.ldap.start_tls|d())|bool)
+                                          else True }}'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# LDAP settings [[[
+# ^^^^^^^^^^^^^^^^^
+
+# .. envvar:: postldap__ldap_user_filter [[[
+#
+# The LDAP filter used to look up user accounts in the directory.
+postldap__ldap_user_filter: '(&
+                                           (objectClass=inetOrgPerson)
+                                           (objectClass=postfixUser)
+                                         )
+                                         (mailEnabled=TRUE)
+                                         (|
+                                           (authorizedService=postfix)
+                                           (authorizedService=mail)
+                                           (authorizedService=\*)
+                                         )'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_domain_filter [[[
+#
+# The LDAP filter used to look up user accounts in the directory.
+postldap__ldap_domain_filter: '(&
+                                            (ObjectClass=dNSDomain)
+                                            (dc=%s)
+                                           )'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_quota_attribute [[[
+#
+# Set the LDAP attribute value to be read by Postfix in order to get the user quota.
+postldap__ldap_quota_attribute: 'mailQuota'
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_quota_default [[[
+#
+# User default LDAP quota. Use human-readable values, e.g. "2 GB".
+postldap__ldap_quota_default: '10 GB'
```

Comment:
Postfix doesn't support quota in a standard installation, there are patches for that but it looks like Debian `postfix` package does not carry one. There is probably a better solution offered by Dovecot - a [policy service](https://blog.sys4.de/postfix-dovecot-mailbox-quota-en.html) which can be queried by Postfix. It might be a better solution in a distributed environment.

Since the quota variables are not used elsewhere in the PR, and they won't be used due to above reasons, I would remove them from the PR entirely. The `mailQuota` attribute might come in handy with Dovecot, but that's not relevant for this PR.